Lib/threading.py

```diff
@@ -568,8 +568,8 @@ class Barrier:
     """Implements a Barrier.
 
     Useful for synchronizing a fixed number of threads at known synchronization
-    points.  Threads block on 'wait()' and are simultaneously once they have all
-    made that call.
+    points.  Threads block on 'wait()' and are simultaneously awoken once they
```

Comment:
I'd use `unblock` instead of `awoken`. I think there's a typo at beginning of the sentence as well. What do you think of `Threads are blocked on...`?

Reply:
I used `awoken` because it is used in the `Barrier.wait` docstring. Would you still prefer `unblock`? And if so, should I change it both places?

I don't see the typo at the beginning of the sentence. English is not my native language, so please show me :)

Reply:
Since the first part says that the threads were blocked, I think make more sense to use "unblock". For consistency, +1 for changing on `Barrier.wait` too.

I'm not a native speaker as well :) It just seems that `Threads block on 'wait()'...` should be `Threads are blocked on 'wait()'...`.

Reply:
To change the terminology in  `threading`, we would have to open a new issue on the tracker. I don't think it has anything to do with this pr. If you think it is an important issue, you can go ahead, I'm not interested.

It will probably end up in a long [red shed](http://red.bikeshed.com/) discussion.